src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Presentation/ImageMonikers.cs

```diff
@@ -0,0 +1,284 @@
+                        switch (GetAccessibility(tags))
+                        {
+                            case Accessibility.Public:
+                                return KnownMonikers.EventPublic;
+                            case Accessibility.Protected:
+                                return KnownMonikers.EventProtected;
+                            case Accessibility.Private:
+                                return KnownMonikers.EventPrivate;
+                            case Accessibility.Internal:
+                                return KnownMonikers.EventInternal;
+                            default:
+                                return KnownMonikers.Event;
+                        }
+
+                    case CompletionTags.ExtensionMethod:
+                        // Extension methods have the same glyph regardless of accessibility.
+                        return KnownMonikers.ExtensionMethod;
+
+                    case CompletionTags.Field:
+                        switch (GetAccessibility(tags))
+                        {
+                            case Accessibility.Public:
+                                return KnownMonikers.FieldPublic;
+                            case Accessibility.Protected:
+                                return KnownMonikers.FieldProtected;
+                            case Accessibility.Private:
+                                return KnownMonikers.FieldPrivate;
+                            case Accessibility.Internal:
+                                return KnownMonikers.FieldInternal;
+                            default:
+                                return KnownMonikers.Field;
+                        }
+
+                    case CompletionTags.Interface:
+                        switch (GetAccessibility(tags))
+                        {
+                            case Accessibility.Public:
+                                return KnownMonikers.InterfacePublic;
+                            case Accessibility.Protected:
+                                return KnownMonikers.InterfaceProtected;
+                            case Accessibility.Private:
+                                return KnownMonikers.InterfacePrivate;
+                            case Accessibility.Internal:
+                                return KnownMonikers.InterfaceInternal;
+                            default:
+                                return KnownMonikers.Interface;
+                        }
+
+                    // TODO: Figure out the right thing to return here.
+                    case CompletionTags.Intrinsic:
+                        return KnownMonikers.Type;
+
+                    case CompletionTags.Keyword:
+                        return KnownMonikers.IntellisenseKeyword;
+
+                    case CompletionTags.Label:
+                        return KnownMonikers.Label;
+
+                    case CompletionTags.Local:
+                        return KnownMonikers.FieldPublic;
```

Comment:
This is not right.  There is something like "KnownMonikers.LocalVariable" for this.